google/api/endpoint.proto

```diff
@@ -45,20 +42,20 @@ message Endpoint {
   // The canonical name of this endpoint.
   string name = 1;
 
+  // Unimplemented. Dot not use.
+  //
   // DEPRECATED: This field is no longer supported. Instead of using aliases,
   // please specify multiple [google.api.Endpoint][google.api.Endpoint] for each of the intended
   // aliases.
   //
   // Additional names that this endpoint will be hosted on.
-  repeated string aliases = 2;
-
-  // The list of features enabled on this endpoint.
-  repeated string features = 4;
```

Comment:
This looks like a breaking change.

Reply:
Great catch! I'll update the release notes.  This breaking change is from https://github.com/googleapis/googleapis/commit/ca1372c6d7bcb199638ebfdb40d2b2660bab7b88. There is a comment that says ``features` field is no longer used. It has been removed from the API runtime for awhile.` Based on the fact that's it's already removed from the API runtime, the breaking change already happened on the backend and existing code would already fail. In this case, we won't bump the major version.